--- /opt/rocks/share/devel/etc/Rules.mk.orig	2017-12-02 01:31:46.000000000 +0000
+++ /opt/rocks/share/devel/etc/Rules.mk	2019-02-14 17:21:48.000000000 +0000
@@ -595,10 +595,10 @@
 	@if [ "$(USERID)" != "0" ] ; then \
 		echo ; \
 		echo ; \
-		echo ERROR - YOU MUST BE ROOT TO BUILD PACKAGES; \
+		echo WARNING - ROCKS MODIFIED TO ALLOW BUILD AS USER WITH SUDO; \
 		echo ; \
 		echo ; \
-		exit 1 ; \
+		exit 0 ; \
 	fi
 
 # --------------------------------------------------------------------- #
