--- /opt/rocks/lib/python2.7/site-packages/rocks/commands/create/roll/__init__.py.orig	2017-12-01 23:51:11.000000000 +0000
+++ /opt/rocks/lib/python2.7/site-packages/rocks/commands/create/roll/__init__.py	2019-02-14 17:21:48.000000000 +0000
@@ -298,11 +298,13 @@
 	def copyRoll(self, roll, dir):
 		tmp = self.mktemp()
 		os.makedirs(tmp)
-		subprocess.call('mount -o loop -t iso9660 %s %s' %
+		# Do not require mount to build rolls...
+		#subprocess.call('mount -o loop -t iso9660 %s %s' %
+		subprocess.call('7z x -y "%s" -o%s' %
 			  (roll.getFullName(), tmp), shell=True)
 		tree = rocks.file.Tree(tmp)
 		tree.apply(self.copyFile, dir)
-		subprocess.call('umount %s' % tmp, shell=True)
+		#subprocess.call('umount %s' % tmp, shell=True)
 		shutil.rmtree(tmp)
 
 
@@ -974,6 +976,7 @@
 	<related>disable roll</related>
 	<related>list roll</related>
 	"""
+	MustBeRoot = 0
 
 	def run(self, params, args):
 		
